runtime/jvmti/jvmtiHook.c

```diff
@@ -1579,18 +1579,15 @@ jvmtiHookGetEnv(J9HookInterface** hook, UDATA eventNum, void* eventData, void* u
 
 	Trc_JVMTI_jvmtiHookGetEnv_Entry();
 
-	if (data->rc == JNI_EVERSION) {
+	if (JNI_EVERSION == data->rc) {
 		jint version = data->version & ~JVMTI_VERSION_MASK_MICRO;
 
 		if ((JVMTI_VERSION_1_0 == version)
 			|| (JVMTI_VERSION_1_1 == version)
 			|| (JVMTI_VERSION_1_2 == version)
 #if JAVA_SPEC_VERSION > 8
-			|| (JVMTI_VERSION_9 == version)
-			|| (JVMTI_VERSION_11 == version)
-#if JAVA_SPEC_VERSION >= 15
-			|| (JVMTI_VERSION_15 == version)
-#endif /* JAVA_SPEC_VERSION >= 15 */
+			|| 	(J9_ARE_ALL_BITS_SET(version, JVMTI_VERSION_INTERFACE_JVMTI) 
```

Comment:
This is not correct for the same reasons as before - please use the snippet I suggested (with the shift added). Also, there seems to be extra spacing here.

Reply:
This is addressed via PR updates (pending reviewing).